### PR TITLE
Add support for recording mode

### PR DESCRIPTION
**Pull request by rayslxu**
This is similar to stanford-oval/almond-server#120 plus stanford-oval/almond-server#123

### CODE REVIEW

File: views/my_stuff.pug [hunk 1]
```diff
@@ -84,6 +84,43 @@ block content
...
+
+  div.modal#recording-warning
+    div.modal-dialog
+      div.modal-content
+        div.modal-body
+          h4= _("Warning:")
+          p
+            = _("While recording is active, all your commands and all the replies from Almond will be stored. ")
+            = _("The recording will include your account and device names, as well as the results from performing actions on your accounts.")
+            = _("The recording will be stored locally and you will have the option to download it.")
```
Comment (gcampax) [inline views/my_stuff.pug:108]: Is it missing a space here between the last two sentences?

**Review by gcampax — APPROVED** [review]
Yeah this all looks sensible. I assume you tested it?
Eventually, we'll want automated tests too, with selenium.

File: views/my_stuff.pug (hunk 1 above)
Comment (rayslxu) [reply views/my_stuff.pug:108]: good catch

**Comment by rayslxu** [comment]
> Yeah this all looks sensible. I assume you tested it?
> Eventually, we'll want automated tests too, with selenium.

I manually tested it. Yeah, eventually I should add automated tests

**Pull request merged**